content/documentation/staging/tutorial/03-first-steps/02-enable-sidecars.adoc

```diff
@@ -0,0 +1,33 @@
+---
+title: Enable Sidecars
+weight: 32
+---
+:icons: font
+:sectlinks:
+
+In this tutorial, we will add namespaces and workloads into the ServiceMesh individually step by step.
+
+This will help users to understand how Istio sidecar proxies work and how applications can use Istio's features.
+
+We are going to start with the *control* workload deployed into the *travel-control* namespace:
+
+NOTE: *[1]* Enable Auto Injection on *travel-control* namespace
```

Comment:
Not sure if it's just me, but seeing numbers in brackets like this sound like footnotes, not steps. I'd suggest something like **1.** instead

Reply:
Perhaps using "[1] -> Step 1", to differentiate of a list.